src/components/Api.js

```diff
@@ -0,0 +1,181 @@
+export default class Api {
+  constructor({ baseURL, authToken }) {
+    this._baseURL = baseURL;
+    this._authToken = authToken;
+  }
+
+  // GET https://around.nomoreparties.co/
+  getCardList() {
+    return fetch("https://around.nomoreparties.co/v1/group-12/cards", {
```

Comment:
It's needed to use `this._baseURL` in all the methods

Reply:
Done